plugins/org.yakindu.sct.doc.user/src/user-guide/code_generation_c.textile

```diff
@@ -14,19 +14,16 @@ h4(#codegen_c_identifiersettings_feature). IdentifierSettings feature
 The *IdentifierSettings* feature allows the configuration of module names and identifier character length:
 
 * _moduleName_ (String, optional): Name for header and implementation. Default: statechart name.
-* _statemachinePrefix_ (Boolean, optional): Prefix that is prepended to function, state, and type names.
-* _maxIdentifierLength_ (Integer, optional): Maximum number of characters of an identifier. Default: 31 characters, which is complying with the ANSI C99 standard.
```

Comment:
We should leave this property with an additional validation stating 'This property is not supported any longer. Please use @ShortCIdentifier annotation in C statecharts. 

This provides feedback to users if they used this property while allowing us to remove the outdated naming service.